src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2749,9 +2783,9 @@ struct MHAHelper {
                                            ov::element::f32,
                                            alibi_slope);
                 if (score_output) {
-                    memcpy(score_output + h * rnd_up(cur_kv_len, 16),
-                           _weight.ptr<float>(ithr, h - hq_beg, pq),
-                           cur_kv_len * sizeof(float));
+                    std::memcpy(score_output + h * rnd_up(cur_kv_len, 16),
```

Comment:
could we add an explanation why do we have 16 here ？ It's better we assign an meaningful name here.

Reply:
The comment is added.